lock-api/src/main/java/com/palantir/lock/v2/LockRequest.java

```diff
@@ -43,15 +43,15 @@
     @Value.Parameter
     Optional<String> getClientDescription();
 
-    static LockRequestV2 of(Set<LockDescriptor> lockDescriptors, long acquireTimeoutMs) {
+    static LockRequest of(Set<LockDescriptor> lockDescriptors, long acquireTimeoutMs) {
         return ImmutableLockRequestV2.of(
                 UUID.randomUUID(),
                 lockDescriptors,
                 acquireTimeoutMs,
                 Optional.of("Thread: " + Thread.currentThread().getName()));
     }
 
-    static LockRequestV2 of(Set<LockDescriptor> lockDescriptors, long acquireTimeoutMs, String clientDescription) {
+    static LockRequest of(Set<LockDescriptor> lockDescriptors, long acquireTimeoutMs, String clientDescription) {
         return ImmutableLockRequestV2.of(
```

Comment:
As before, will need to change this to just `ImmutableLockRequest` (and in the other `of()` too)